components/raftstore/src/store/hibernate_state.rs

```diff
@@ -95,14 +108,67 @@ impl HibernateState {
             LeaderState::Poll(v) => v,
             LeaderState::Hibernated => return true,
         };
+        hib_vote_peer_ids.extend(v.iter());
+        let hib_vote_peer_ids_str = format!("{:?}", hib_vote_peer_ids);
```

Comment:
Please clean these format str, they can be formatted automatically by `"xxx" => ?hib_vote_peer_ids`